addons/sale/controllers/product_configurator.py

```diff
@@ -5,10 +5,10 @@
 from odoo.http import Controller, request, route
 
 
-class ProductConfiguratorController(Controller):
+class SaleProductConfiguratorController(Controller):
 
-    @route('/sale_product_configurator/get_values', type='json', auth='user')
-    def get_product_configurator_values(
+    @route(route='/sale_product_configurator/get_values', type='json', auth='user')
```

Comment:
If I'm not mistaken, RPCs are always sent via POST, so this will probably not work. I'll double-check to make sure.

The reason I added `methods=['POST'],` elsewhere is because exposing a GET endpoint for methods that modify data poses a security risk (people may try to forge requests to said endpoint).